plugins/woocommerce/includes/gateways/paypal/class-wc-gateway-paypal.php

```diff
@@ -17,6 +17,8 @@
 	exit;
 }
 
+require_once __DIR__ . '/includes/class-wc-gateway-paypal-request.php';
```

Comment:
Should we add a `class_exists()` safety check here as well, and remove the `include_once ...` inside `process_payment()`?